frameworks/elastic/README.md

```diff
@@ -26,13 +26,19 @@ DC/OS Elasticsearch provides the following features:
 
 # Quick Start
 
-1. Install an Elasticsearch cluster and log on.
+1. Install an Elasticsearch cluster with Kibana and log on to the Mesos master node.
 
   ```bash
   dcos package install --app elastic
   dcos node ssh --master-proxy --leader
   ```
 
+1. Wait until the cluster is deployed and the nodes are all running. This may take 5-10 minutes. If you try to access the cluster too soon, you may get an empty response or an authentication error like this:
```

Comment:
In fact, plan completeness probably doesn't signal that Kibana is ready to go right now.  It looks like the health-check should be changed to a readiness check or replicated as a readiness check, as it's testing whether the login endpoint is ready yet.  Killing Kibana if it fails to be ready within the given timeout may or may not be a good idea.